e2e/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "radicle-local-network"
```

Comment:
Would it be advised to encode `link` here to avoid potential name clashes with other local network tooling. a la `radicle-link-local-network`.

Reply:
While on it, is it planned to be released as a crate?

Reply:
No. `cargo` doesn't publish the entire workspace unattended, does it?